src/main/java/org/broadinstitute/hellbender/engine/LocusWalker.java

```diff
@@ -177,4 +211,26 @@ protected final void onShutdown() {
         // Overridden only to make final so that concrete tool implementations don't override
         super.onShutdown();
     }
+
+    /**
+     *  The emit empty loci parameter comes with several pitfalls when used incorrectly.  Here we check and either give
+     *   warnings or errors.
+     */
+    protected void validateEmitEmptyLociParameters() {
+        if (emitEmptyLoci()) {
+            if (getBestAvailableSequenceDictionary() == null) {
+                throw new UserException.MissingReference("Could not create a sequence dictionary nor find a reference.  Therefore, emitting empty loci is impossible and this tool cannot be run.  The easiest fix here is to specify a reference dictionary.");
+            }
+            if (!hasReference() && !hasIntervals()) {
+                logger.warn("****************************************");
```

Comment:
I do not believe that the missing reference one is presumptuous.  If you cannot get a sequence dictionary of any kind and you have no intervals, then you can't determine the loci to traverse if you are emitting uncovered loci.

As for the second one (``!hasReference() && !hasIntervals()``) this means that the GATK created a dictionary from some other source.  The user would have to be pretty advanced to know how that would play out.  

We can discuss more, but I would like to leave this one in.  Especially, since I think most users using emit empty loci will have forgotten their interval file and/or reference if this warning gets triggered.